pkg/graph/schema/types/mto_milestone.graphql

```diff
@@ -52,22 +52,26 @@ input MTOMilestoneChanges @goModel(model: "map[string]interface{}") {
   isDraft: Boolean
 }
 
+input MTOSolutionLinks {
+  solutionIDs: [UUID!]
+  commonSolutionKeys: [MTOCommonSolutionKey!]
+}
+
 extend type Mutation {
   createMTOMilestoneCustom(modelPlanID: UUID!, name: String!, mtoCategoryID: UUID): MTOMilestone!
   @hasRole(role: MINT_USER)
   createMTOMilestoneCommon(modelPlanID: UUID!, commonMilestoneKey: MTOCommonMilestoneKey!, commonSolutions: [MTOCommonSolutionKey!]): MTOMilestone!
   @hasRole(role: MINT_USER)
-  updateMTOMilestone(id: UUID!, changes: MTOMilestoneChanges!): MTOMilestone!
+  updateMTOMilestone(id: UUID!, changes: MTOMilestoneChanges!, solutionLinks: MTOSolutionLinks): MTOMilestone!
   @hasRole(role: MINT_USER)
   deleteMTOMilestone(id: UUID!): Boolean!
   @hasRole(role: MINT_USER)
+  mtoMilestoneUpdateLinkedSolutions(id: UUID!, solutionLinks: MTOSolutionLinks!): MTOMilestone!
```

Comment:
Maybe this should return an array of Solutions instead?

Reply:
I'd be fine either way, honestly. Returning the milestone could even be preferred since the `id` we're passing in is for the milestone, so in a sense, it's the primary object being operated on!

This is also, more likely than not, another situation where the FE isn't even using the returned type, so I wouldn't worry too much about this in all honesty 😅